app/src/main/res/xml/sponsorblock_settings.xml

```diff
@@ -33,64 +33,64 @@
             app:summary="@string/category_selfpromo_description"
             app:title="@string/category_selfpromo"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="manual"/>
+            app:defaultValue="@string/manual"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="interaction_category_key"
             app:summary="@string/category_interaction_description"
             app:title="@string/category_interaction"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="intro_category_key"
             app:summary="@string/category_intro_description"
             app:title="@string/category_intro"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="outro_category_key"
             app:summary="@string/category_outro_description"
             app:title="@string/category_outro"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="filler_category_key"
             app:summary="@string/category_filler_description"
             app:title="@string/category_filler"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="music_offtopic_category_key"
             app:summary="@string/category_music_offtopic_description"
             app:title="@string/category_music_offtopic"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
 
-        <DropDownPreference
+        <com.github.libretube.ui.views.SpinnerPreference
             app:key="preview_category_key"
             app:summary="@string/category_preview_description"
             app:title="@string/category_preview"
             app:entries="@array/sb_skip_options"
             app:entryValues="@array/sb_skip_options_values"
-            app:defaultValue="off"/>
+            app:defaultValue="@string/off"/>
```

Comment:
Please don't use translatable strings to store preference values. If the translation is getting changed, the preference value would also change and not work anymore.

Reply:
Hmm I see, when I put the string in directly it breaks the code. Do you know why this might be happening?
<img width="165" alt="Screen Shot 2023-06-18 at 4 30 27 PM" src="https://github.com/libre-tube/LibreTube/assets/48372480/db49cc36-65e2-4c82-8968-ac77cec20a4d">